packages/shared/src/components/MainLayout.tsx

```diff
@@ -47,6 +46,11 @@ const GoBackHeaderMobile = dynamic(
   { ssr: false },
 );
 
+const Sidebar = dynamic(
+  () => import(/* webpackChunkName: "sidebar" */ './sidebar/Sidebar'),
+  { ssr: false },
```

Comment:
This replaced hydration hack right?

Reply:
I have to test, I think its gonna delay sidebar a bit more due to seaprate bundle and JS needs to download, but I think its actually ok.

Reply:
Ah, this is just to not have it in the same bundle, but in actual replacement of the hack, literally any fetch status is something we can use. On this one, we used the `isAuthReady`.

Reply:
Looks good, in the future we would also be able to use Suspense here with potentially even server component.. but thats for some other day.